src/gui/remote/RemoteHandler.cpp

```diff
@@ -67,7 +67,7 @@ RemoteHandler::RemoteResult RemoteHandler::download(const RemoteParams* params)
             remoteProcess->closeWriteChannel();
         }
 
-        bool finished = remoteProcess->waitForFinished(10000);
+        bool finished = remoteProcess->waitForFinished(params->downloadTimeout * 1000);
```

Comment:
Instead of doing math here I recommend doing that at the setting of the downloadTimeout (which could be called `downloadTimeoutMsec` to be explicit). In general, I am trying to keep all "display math" at the GUI layer and the backend has well defined parameters.